.github/workflows/deploy-apis-to-production.yml

```
@@ -1875,4 +1883,35 @@ jobs:
+        with:
+          registry: ${{ env.REGISTRY_URL }}
+          username: _json_key
+          password: ${{ secrets.GCR_CONFIG }}
+
+      - name: Login to K8S
+        uses: azure/k8s-set-context@v3.0
+        with:
+          method: kubeconfig
+          kubeconfig: ${{ secrets.K8S_CONFIG_PROD }}
+
+      - name: Deploy Redis via Helm 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)